### ISSUE TITLE
Piling

### ISSUE THREAD

**Issue by yurigangkor** [S-101-Portrayal-subWG/Working-Documents#53]
[9. Piling.zip](https://github.com/S-101-Portrayal-subWG/Working-Documents/files/8204325/9.Piling.zip)
![Piling](https://user-images.githubusercontent.com/92765146/157202116-e9959285-bed9-4c3d-a55d-63c1b09bd232.JPG)

**Comment by IngaFjellanger** [comment]
In xml file the offered line pattern describes a repeated pattern, 2 dashes and 2 PILPNT02 symbols. The line pattern could be simplified like this:
 ```
<lineStyle offset="0">
 <intervalLength>12</intervalLength>
  <pen width="0.32">
    <color>CHGRD</color>
  </pen>
  <dash>
    <start>0</start>
    <length>6</length>
  </dash>
  <symbol reference="PILPNT02">
    <position>9</position>
  </symbol>
</lineStyle>
```

**Comment by SylviaSpohn-BSH** [comment]
Agree to the line symbol, but could it be mixed up with log pond INT1 F29.1 on paper charts?

**Comment by DavidGrant-NIWC** [comment]
Related: #51

**Comment by alvarosanuy** [comment]
**Decision made at Portrayal subWG meeting on 14/7/22**
This **categoryOfPile** should only be used for geometric primitive Curve and Point (as per DCEG extract below)

![image](https://user-images.githubusercontent.com/70189016/180099848-1e8d69c5-6803-43ea-bf34-bf92757e6548.png)

Despite this, portrayal implementation below recommends (just in case) linking Surface as well (which will effectively display as 'Area of Piles' - **categoryOfPile**=6).

1. Approve implementation of complex line style as uploaded by KHOA to this Github space.
**NIWC** to review the encoding of xml and consider Ingas' input above (24/5) as an improvement to the xml file for the curve.
2. Use new complex line for Curves and Surfaces (_Plain_ and _Symbolised_ boundaries).
3. For Point geometry use existing S-52 symbol (PILPNT02).
4. Update rules accordingly [**Piles**; **categoryOfPile**=5 (piling)]

**Comment by yurigangkor** [comment]
![Piling](https://user-images.githubusercontent.com/92765146/180146342-ea6487d9-2a25-40c1-9310-33098925b948.PNG)
[Piling.zip](https://github.com/S-101-Portrayal-subWG/Working-Documents/files/9156802/Piling.zip)

**Comment by alvarosanuy** [comment]
This issue has been re opened until changes are implemented in PC 1.0.2 by NIWC.

**Comment by DavidGrant-NIWC** [comment]
See #51

**Comment by alvarosanuy** [comment]
Implemented in PC 1.0.2
